baseapp_chats/triggers.py

```
@@ -2,8 +2,8 @@
 
 
 class Func(pgtrigger.Func):
-    def render(self, meta=None, fields=None, columns=None, **kwargs) -> str:
-        return self.func.format(model=meta.model, meta=meta, fields=fields, columns=columns)
+    def render(self, model=None, fields=None, columns=None, **kwargs) -> str:
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)